libfwupd/fwupd-enums.h

```diff
@@ -539,6 +539,78 @@ typedef enum {
  */
 typedef guint64 FwupdDeviceFlags;
 
+/**
+ * FWUPD_DEVICE_PROBLEM_NONE:
+ *
+ * No device problems detected.
+ *
+ * Since 1.8.1
+ */
+#define FWUPD_DEVICE_PROBLEM_NONE (0u)
+/**
+ * FWUPD_DEVICE_PROBLEM_SYSTEM_POWER_TOO_LOW:
+ *
+ * The system power is too low to perform the update.
+ *
+ * Since 1.8.1
+ */
+#define FWUPD_DEVICE_PROBLEM_SYSTEM_POWER_TOO_LOW (1u << 0)
+/**
+ * FWUPD_DEVICE_PROBLEM_UNREACHABLE:
+ *
+ * The device is unreachable, or out of wireless range.
+ *
+ * Since 1.8.1
+ */
+#define FWUPD_DEVICE_PROBLEM_UNREACHABLE (1u << 1)
```

Comment:
Thinking forward this can be something to use for transiently connected USB devices too. Like if there is an update available for a dongle connected to your work dock only.

Reply:
Yup, but there be dragons -- in that you need a way to "forget" devices like this too.